lib/msf/core/exploit/remote/http_server/php_include.rb

```diff
@@ -62,7 +62,7 @@ def exploit
     rescue ::Interrupt
       raise $!
     ensure
-      stop_service
+      cleanup_service
```

Comment:
Can't see this needed either, given the exploit driver does this:

https://github.com/rapid7/metasploit-framework/blob/c3465a8ad8369ef2eab3acd9d1a21b19feed4d3e/lib/msf/core/exploit_driver.rb#L180-L190